db/migrate/20200129122439_copy_cloud_tenants_ems_ref_to_tenants.rb

```diff
@@ -0,0 +1,36 @@
+class CopyCloudTenantsEmsRefToTenants < ActiveRecord::Migration[5.1]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Tenant < ActiveRecord::Base; end
+
+  class CloudTenant < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Copying CloudTenant#ems_ref or ExtManagementSystem#guid to Tenant#ems_ref") do
+      Tenant.where.not(:source_id => nil).each do |tenant|
+        case tenant.source_type
+        when "CloudTenant"
+          source_scope = CloudTenant.where(:id => tenant.source_id)
```

Comment:
Is it possible for there to be more than one CloudTenant record with the same tenant.source_id?  This looks like it should just be `CloudTenant.find_by`